test/shared/utils.py

```diff
@@ -66,37 +66,17 @@ def cibuildwheel_run(project_path, env=None, add_env=None, output_dir=None):
     return wheels
 
 
-def expected_wheels(package_name, package_version):
+def expected_wheels(package_name, package_version, manylinux_versions=['manylinux1', 'manylinux2010']):
```

Comment:
Why is the default both 'manylinux1' and 'manylinux2010'? I thought the default was to build `manylinux2010` only?

Reply:
Our simple, default `spam.c` is so simple that even when we build it on a `manylinux2010` image (where it can use more recent versions of `glibc` etc.), it produces a shared library that's also still `manylinux1` compatible.

Reply:
Whoops. Ah yes. Realised this last night after I left this comment!